index.html

```diff
@@ -333,19 +348,39 @@ <h3>Degradation preference when encoding</h3>
             <tr>
               <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl>maintain-framerate</dfn></td>
               <td>
-                <p>Degrade resolution in order to maintain framerate.</p>
+                <p>Degrade resolution in order to maintain framerate. The user
+                agent SHOULD prefer reducing the resolution in order to optimize
+                for video quality and performance within network constraints.</p>
               </td>
             </tr>
             <tr>
               <td><dfn data-idl>maintain-resolution</dfn></td>
               <td>
-                <p>Degrade framerate in order to maintain resolution.</p>
+                <p>Degrade framerate in order to maintain resolution. The user
+                agent SHOULD prefer reducing the framerate in order to optimize
+                for video quality and performance within network constraints.</p>
               </td>
             </tr>
             <tr>
               <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl>balanced</dfn></td>
               <td>
-                <p>Degrade a balance of framerate and resolution.</p>
+                <p>Degrade a balance of framerate and resolution. The user agent
+                SHOULD prefer reducing a balance of framerate and resolution in
+                order to optimize for video quality and performance within
+                network constraints.</p>
+              </td>
+            </tr>
+            <tr>
+              <td data-tests="RTCRtpParameters-degradationPreference.html"><dfn data-idl>maintain-framerate-and-resolution</dfn></td>
+              <td>
+                <p>
+                  Prefer to maintain framerate and resolution even if this
+                  requires adjusting encoding parameters to yield a suboptimal
+                  video quality. The user agent SHOULD NOT prefer reducing the
+                  framerate or resolution for quality and performance reasons,
+                  but MAY drop frames before encoding if necessary not to
+                  overuse network and encoder resources.
+                </p>
```

Comment:
Doesn't "MAY drop frames" actually results in maintaining resolution over frame-rate?